dts/arm/st/h7/stm32h753.dtsi

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2020 Moonkwun Jung
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <st/h7/stm32h7.dtsi>
+
+/ {
+	cpus {
+		/delete-node/ cpu@1;
+	};
+
+	/* 512KB SRAM0 @ 24000000 */
+	sram0: memory@24000000 {
+		compatible = "mmio-sram";
+		reg = <0x24000000 DT_SIZE_K(512)>;
+	};
+
+	/* 128KB SRAM1 @ 30000000 */
+	sram1: memory@30000000 {
+		compatible = "mmio-sram";
+		reg = <0x30000000 DT_SIZE_K(128)>;
+	};
+
+	/* 128KB SRAM2 @ 30020000 */
+	sram2: memory@30020000 {
+		compatible = "mmio-sram";
+		reg = <0x30020000 DT_SIZE_K(128)>;
+	};
+
+	/* 32KB SRAM3 @ 30040000 */
+	sram3: memory@30040000 {
+		compatible = "mmio-sram";
+		reg = <0x30040000 DT_SIZE_K(32)>;
+	};
+
+	/* 64KB SRAM4 @ 38000000 */
+	sram4: memory@38000000 {
+		compatible = "mmio-sram";
+		reg = <0x38000000 DT_SIZE_K(64)>;
+	};
+
+	/* 4KB BackupSRAM @ 38800000 */
+	Backupsram4: memory@38800000 {
+		compatible = "mmio-sram";
+		reg = <0x38800000 DT_SIZE_K(4)>;
+	};
```

Comment:
Due to lack of support in linker, these sections are not accessible for zephyr application (in current master).
I see no value in adding them right now, due to the confusion it could create.

Reply:
What code should I modify before adding sram content?

Reply:
First question: do you intent to use all these sections?
Adding this is not a simple task. `soc/arm/nxp_imx/rt/linker.ld` could be an example of what has to be done, but it will likely have other impacts.